R/lazyframe-frame.R

```diff
@@ -1749,7 +1749,7 @@ lazyframe__rolling <- function(
 #' - 1i # length 1
 #' - 10i # length 10
 #'
-#' @return A [LazyGroupBy][LazyGroupBy_class] object
+#' @return A [LazyGroupByDynamic][LazyGroupByDynamic_class] object
```

Comment:
Not sure if we should have the same class for eager and lazy

Reply:
IIUC, these are not distinguished on the LazyFrame (i.e. Rust side), so it is not worth distinguishing them on the Python/R side.

Reply:
I've put `GroupByDynamic` in `@return` for eager and lazy